src/rust/ide/lib/view/src/graph_editor.rs

```diff
@@ -2306,6 +2341,31 @@ fn new_graph_editor(app:&Application) -> GraphEditor {
 
     }
 
+    // === Documentation Set ===
+    frp::extend! { network
+
+    def _set_doc_data = inputs.set_documentation_data.map(f!((data) {
+         model.doc_view.frp.send_data.emit(data);
+     }));
```

Comment:
1. alignment (here and other places)
2. Remove curly braces and semicolon, not needed.